README.md

```diff
@@ -127,7 +104,7 @@ package = "@netlify/plugin-nextjs"
 
 ## Manually upgrading from an older version of the Next.js Runtime
 
-If you previously set these values, they're no longer needed and can be removed:
+If you previously set these values, they're no longer needed and should be removed:
 
 - `distDir` in your `next.config.js`
 - `node_bundler = "esbuild"` in `netlify.toml`
```

Comment:
What about NEXT_USE_NETLIFY_EDGE in the toml?

Reply:
Ah, good addition